### ISSUE TITLE
Wheres Wakanda ? 

### ISSUE THREAD

**Issue by erlicthemad** [Wakanda/wakanda-issues#259]
I was trying to figure out why my copy of Wakanda kept giving me the beachball of death and installed Little Snitch to see what Wakanda was doing.  I was figuring that Wakanda was trying to get to something on the network and timing out a DNS or similar lookup while waiting. This is a common cause for lots of spinning beach balls.  What I discovered was very surprising. Why is my copy of Wakanda checking in with online trackers and analytics? Why is my copy of Wakanda when I just open a local project talking to 10 different online resources. 

![image](https://user-images.githubusercontent.com/868472/41238236-73c61df2-6d63-11e8-89ec-26c22f2e7d45.png)

Which all these connections turned off I still get the beachball of death running as I try to do anything in the program. Why does my local program need to consult with 10 different cloud-based resources just to edit a text file? Does this have anything to do with why Wakanda is such a slow editor?

RIght now the Wakanda editor is useless since I can only type on average 2 characters before I get a spinning beach ball for 30 - 45 seconds.

**Comment by erlicthemad** [comment]
After blocking all the network connections I found that many of my pinwheel errors stopped happening.  Or when they did the delay was much shorter.

**Comment by JegoYann** [comment]
Hello Elrick,

Indeed, the studio does have some requests in its home page. You can close it and it won't be open again.
Did you see something else after closing it ?

Regards,
Yann

**Comment by erlicthemad** [comment]
What I noticed is that after shutting them down the pinwheel of death instances slowed down.  Since I don't have any debugging info. I am not sure why the editor stop what it's doing and spins the pinwheel waiting for some kind of resource before returning.

**Comment by hamzahik** [comment]
@erlicthemad please try the following scenario to confirm it is related to the "start page" / analytics : 
- close studio
- disconnect your computer from the internet (no wifi no ethernet)
- open the studio
- try to reproduce the issue

**Comment by erlicthemad** [comment]
I just did your cycle. I am now running with the new 2.6.0 Enterprise. I still get the spinning pinwheel of death when I try to edit files. The file I am trying to edit is a whopping 55 lines long.  I type 5 characters and off goes the pinwheel.

**Comment by erlicthemad** [comment]
I know my computer should not be having these issues. I am only running an Intel Core i7 with 16 GB of ram and a 512GB SSD.  My external work storage drive is a 1TB SSD.  With any of my other IDEs its snappy and quick, xCode, Zend Studio, and Visual Studio Code, and even BBEdit.

**Comment by JegoYann** [comment]
@erlicthemad Is it possible to capture a screen video of your issue ?
Looks like you are mixing two issues. One with the Home page sending analytics request. One with the code editor and a spinning pinwheel that we are not aware of.

**Comment by JegoYann** [comment]
@erlicthemad Thx you can remove the link if you want. We will take a look.

**Comment by JegoYann** [comment]
@erlicthemad Looks like a autocompletion issue.
Could you deactivate the code editor autocomplete and try again ?
Do you reproduce the issue ?

File > Preferences > Code Editor > Javascript preferences > Display autocomplete suggestions > on demand

![screen shot 2018-06-13 at 17 15 51](https://user-images.githubusercontent.com/3337770/41360717-b02c6fb4-6f2d-11e8-8c83-81035a8f3bdc.png)

**Comment by erlicthemad** [comment]
No, I just restarted Wakanda after quitting. Then opened my project and added test 3. 

 
```
 // Test 3.
 var myDate3 = new Date();
 console.log (myDate3.toString());
```
During the typing it still stopped and spun, It also spun after I typed myDate3 with var misspelled as vat.

Here is a picture of my settings after restarting Wakanda. 
<img width="750" alt="screenshot 2018-06-13 11 43 21" src="https://user-images.githubusercontent.com/868472/41362321-14c04290-6eff-11e8-936a-f72c89717a2f.png">

**Comment by JegoYann** [comment]
Thanks for your feedbacks. You did reproduce the issue. We will tackle it.
